net-vpn/tor/tor-0.3.2.9.recipe

```diff
@@ -0,0 +1,99 @@
+SUMMARY="A virtual layer that helps you improve privacy and anonymity"
+DESCRIPTION="Tor is a network of virtual tunnels that allows people and groups to improve \
+their privacy and security on the Internet. It also enables software \
+developers to create new communication tools with built-in privacy features. \
+Tor provides the foundation for a range of applications that allow \
+organizations and individuals to share information over public networks \
+without compromising their privacy."
+HOMEPAGE="https://www.torproject.org"
+COPYRIGHT="
+	2001-2004, Roger Dingledine
+	2004-2006, Roger Dingledine, Nick Mathewson
+	2007-2018, The Tor Project, Inc.
+	"
+LICENSE="Tor"
+REVISION="1"
+SOURCE_URI="https://www.torproject.org/dist/tor-$portVersion.tar.gz"
+CHECKSUM_SHA256="435a7b91aa98d8b1a0ac1f60ca30c0ff3665b18a02e570bab5fe27935829160f"
+PATCHES="tor-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+# On x86_gcc2 we don't want to install the commands in bin/<arch>/, but in bin/.
+commandSuffix=$secondaryArchSuffix
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+	commandSuffix=
+	commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	tor = $portVersion
+	cmd:tor$commandSuffix
+	cmd:tor_gencert$commandSuffix
+	cmd:tor_resolve$commandSuffix
+	cmd:torify$commandSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libcrypto$secondaryArchSuffix
+	lib:libevent_2.1$secondaryArchSuffix
+	lib:liblzma$secondaryArchSuffix
+	lib:libssl$secondaryArchSuffix
+	lib:libz$secondaryArchSuffix
+	lib:libzstd$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libevent_2.1$secondaryArchSuffix
+	devel:liblzma$secondaryArchSuffix
+	devel:libssl$secondaryArchSuffix
+	devel:libz$secondaryArchSuffix
+	devel:libzstd$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:gcc$secondaryArchSuffix
+	cmd:libtoolize$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config$secondaryArchSuffix
+	"
+
+USER_SETTINGS_FILES="
+	settings/tor directory
+	"
+
+GLOBAL_WRITABLE_FILES="
+	settings/tor/torrc.sample auto-merge
+	"
+
+defineDebugInfoPackage tor$secondaryArchSuffix \
+	"$commandBinDir"/tor \
+	"$commandBinDir"/tor-gencert \
+	"$commandBinDir"/tor-resolve
+
+BUILD()
+{
+	export CFLAGS="-D_BSD_SOURCE"
+	export CXXFLAGS="-D_BSD_SOURCE"
+	export LDFLAGS="-lbsd"
+
+	autoreconf -vfi
+
+	runConfigure --omit-dirs "binDir sbinDir" ./configure \
+		--disable-gcc-hardening
```

Comment:
I'd recommend this tiny change:
```diff
 	runConfigure --omit-dirs "binDir sbinDir" ./configure \
+		--bindir="$commandBinDir" --sbindir="$commandBinDir" \
 		--disable-gcc-hardening
```
This is not required _at the moment_, but when `x86` becomes available as a secondary arch for `x86_64` the recipe would not be strictly correct without this change.